pkg/ccl/changefeedccl/changefeedbase/options.go

```diff
@@ -160,3 +160,13 @@ var ChangefeedOptionExpectValues = map[string]sql.KVStringOptValidate{
 	OptWebhookClientTimeout:     sql.KVStringOptRequireValue,
 	OptOnError:                  sql.KVStringOptRequireValue,
 }
+
+// CaseInsensitiveOpts options which supports case Insensitive value
+var CaseInsensitiveOpts = map[string]struct{}{
```

Comment:
I wonder if the option names (keys) should be case insensitive. @amruss?

Reply:
think she is out this week